components/instructions/instructionPanel.tsx

```diff
@@ -78,22 +92,28 @@ export function InstructionPanel() {
             <Disclosure.Panel
               className={`border border-fgd-4 border-t-0 p-4 md:p-6 pt-0 rounded-b-md`}
             >
-              {Object.values(instructions)
-                .sort(
-                  (i1, i2) =>
-                    i1.account.instructionIndex - i2.account.instructionIndex
-                )
-                .map((pi, idx) => (
-                  <div key={pi.pubkey.toBase58()}>
-                    {proposal && (
-                      <InstructionCard
-                        proposal={proposal}
-                        index={idx + 1}
-                        proposalInstruction={pi}
-                      ></InstructionCard>
-                    )}
-                  </div>
-                ))}
+              {proposalInstructions.map((pi, idx) => (
+                <div key={pi.pubkey.toBase58()}>
+                  {proposal && (
+                    <InstructionCard
+                      proposal={proposal}
+                      index={idx + 1}
+                      proposalInstruction={pi}
+                    ></InstructionCard>
+                  )}
+                </div>
+              ))}
+
+              {proposal && (
```

Comment:
We show the "execute all instructions" button for every proposal, even if there's only one instruction ?

Reply:
was displaying even when only one itx. fixed.